### PR TITLE
@dblandin => [SearchResults] Add send feedback component, for logged in users only

### PR DESCRIPTION
**Pull request by mzikherman**
Looks like:

Screen Shot 2019-03-21 at 6 52 28 PM

and on the artworks tab:

Screen Shot 2019-03-21 at 6 52 11 PM

This works for logged in users!

Still to do:
  - ~~when user is logged out, there's additional fields for name/email~~
  - ~~proper success/error handling~~
  - specs

### CODE REVIEW

**Comment by artsyit** [comment]
Deploy preview for *artsy-reaction* ready!

Built with commit bb76b17f6dd62bbeb3ea424ca319f38342492b9c

https://deploy-preview-2201--artsy-reaction.netlify.com

**Comment by mzikherman** [comment]
Here's how the logged out state looks:

Screen Shot 2019-03-21 at 7 26 41 PM

**Comment by mzikherman** [comment]
Added validation for the form fields, and a 'success' state:

search

**Comment by mzikherman** [comment]
@dblandin This is ready to go! I'm working on adding some specs, but should be reviewable/mergeable.

File: src/Apps/Search/Components/SendFeedback.tsx [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+    )
+  }
+}
+
+const FeedbackContainer = styled(Flex)`
+  min-height: 212px;
+`
+
+const FeedbackTextAreaContainer = styled(Box)`
+  width: 484px;
```
[the remark this answers is not in the document]
Comment (mzikherman) [reply src/Apps/Search/Components/SendFeedback.tsx]: I actually specifically didn't inline them so as to 'call them out'. In that, (based on the comps), these are the sizes for the various containers in the layout (which are fixed). So it's easy to see that and make changes.

**Comment by mzikherman** [comment]
> Have if you considered utilizing Formik to reduce some of the state management / boilerplate often involved in form creation?

Good question! I didn't think we really needed any other abstraction here, this wound up being pretty simple (so far!). If it starts getting too hairy I'm definitely open to try to switch this over!

I got curious what was used in the order app (there's a good amount of forms/inputs in that flow), doesn't look like it was used there. So I think there's not _necessarily_ a strict pattern for this (yet), so I tend to like to keep things simple and build up from there. If the behavior here starts to get complex I'll take another look at using it 👍 .

**Comment by artsyit** [comment]
:rocket: PR was released in v15.9.0 :rocket:

**Pull request merged**